proxy.ts

```diff
@@ -320,6 +320,7 @@ const authFlowRedirect = (
       !session.user.acceptableUse &&
       !onSupport &&
       !path.startsWith("/auth/policy") &&
+      path !== "/form-builder" &&
```

Comment:
We may have to dig in a little deeper as to why the redirect was not working properly as this addition would allow a user to bypass the acceptable use page by manually entering the form-builder url or by using a saved bookmark.